Pod/Classes/PINAlternateRepresentationDelegate.h

```diff
@@ -0,0 +1,25 @@
+//
+//  PINAlternateRepresentationDelegate.h
+//  Pods
+//
+//  Created by Garrett Moon on 3/17/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "PINRemoteImageManager.h"
+
+@protocol PINRemoteImageManagerAlternateRepresentationDelegate <NSObject>
+@required
+
+/**
+ @discussion This method will be called with data off the wire or stored in the cache. Return an object to have it returned as the alternativeRepresentation object in the PINRemoteImageManagerResult. @warning this method can be called on the main thread, be careful of doing expensive work.
+ */
+- (id)alternateRepresentationWithData:(NSData *)data options:(PINRemoteImageManagerDownloadOptions)options;
```

Comment:
I assume you have considered this, but there's no way we can be more specific about the return of this method? Don't we need to conform to anything or be of anything? Can it literally be anything?